detection_rules/cli_utils.py

```diff
@@ -187,7 +185,8 @@ def rule_prompt(path=None, rule_type=None, required_only=True, save=True, verbos
 
             contents[name] = result
 
-    suggested_path = os.path.join(RULES_DIR, contents['name'])  # TODO: UPDATE BASED ON RULE STRUCTURE
+    # DEFAULT_PREBUILT_RULES_DIRS[0] is a required directory
+    suggested_path = os.path.join(DEFAULT_PREBUILT_RULES_DIRS[0], contents['name'])
```

Comment:
Might was well update the path below to use `Path`

Reply:
Based on what we decided. Might make more sense to do some of this in https://github.com/elastic/detection-rules/pull/3659

Reply:
Agreed.